boards/xtensa/intel_adsp_ace15_mtpm/intel_adsp_ace15_mtpm.yaml

```diff
@@ -9,5 +9,34 @@ supported:
   - dma
 testing:
   ignore_tags:
+    - cmsis_dsp
+    - cmsis_rtos
+    - zdsp
```

Comment:
`zdsp` is duplicated.

Reply:
I wanted to doubly ensure it was ignored